requirements.txt

```diff
@@ -0,0 +1,5 @@
+dnspython
+matplotlib
+ocflib
```

Comment:
Given this will now depend on `ocflib`, it's probably good to have the ocfweb build be an upstream build trigger to this repo so that this gets rebuilt when ocflib is updated.

Just for reference, this repo is a upstream trigger for the docker repo, since utils are included in each docker image, so building this repo (or ocflib) will cause quite a cascade.